cl/search/tasks.py

```diff
@@ -355,6 +376,12 @@ def update_child_documents_by_query(
     if es_document is PositionDocument:
         s = s.query("parent_id", type="position", id=parent_instance.pk)
         main_doc = PersonDocument.get(id=parent_instance.pk)
+    elif es_document is ESRECAPDocument:
+        s = s.query("parent_id", type="recap_document", id=parent_instance.pk)
+        main_doc = DocketDocument.get(id=parent_instance.pk)
+
+    if not main_doc:
+        return
```

Comment:
We need to remove these lines of code because we already have the if-statement :)